src/rules/function-disallowed-list/README.md

```diff
@@ -0,0 +1,60 @@
+# function-disallowed-list
+
+Specify a list of disallowed functions.
+
+<!-- prettier-ignore -->
+```css
+a { margin-left: math.random(100); }
+/**              ↑
+* This function */
+```
+
+## Options
+
+`array|string|regex`: `["array", "of", "unprefixed", /functions/, "regex"]|"function"|"/regex/"|/regex/`
+
+If a string is surrounded with `"/"` (e.g. `"/^rgb/"`), it is interpreted as a regular expression.
+
+Given:
+
+```json
+["math.random", "double"]
```

Comment:
I forgot to update this line. This is an example of my first approach, which was disallowing a function within a module or any function in the global namespace. After discussing with the Sass team, we concluded it would be better to take in only the function's name and disallow all functions with this name (global or imported), since the module can be imported with any namespace (Eg. `@use "sass:math" as foo`).